6-appsource/cymbal-bank/frontend/k8s/overlays/production/gateway.yaml

```diff
@@ -31,6 +29,10 @@ spec:
     allowedRoutes:
       kinds:
       - kind: HTTPRoute
+    tls:
+      mode: Terminate
+      options:
+        networking.gke.io/pre-shared-certs: frontend-example-com
```

Comment:
how is frontend.example.com converted to frontend-example-com ?

Reply:
Yeah, it should be the `name` of the `google_compute_managed_ssl_certificate` which is `key` value from `certificates`.